src/main/java/ch/njol/skript/sections/EffSecSpawn.java

```diff
@@ -116,12 +119,17 @@ public boolean init(Expression<?>[] exprs,
 		locations = Direction.combine((Expression<? extends Direction>) exprs[1 + matchedPattern], (Expression<? extends Location>) exprs[2 + matchedPattern]);
 
 		if (sectionNode != null) {
-			if (!BUKKIT_CONSUMER_EXISTS) {
-				Skript.error("The spawn section isn't available on your Minecraft version, use a spawn effect instead");
+			AtomicBoolean delayed = new AtomicBoolean(false);
+			trigger = loadCode(
+				sectionNode,
+				"spawn",
+				() -> delayed.set(getParser().getHasDelayBefore().isTrue()),
```

Comment:
You shouldn't use isTrue, it will for example not trigger for
```
spawn ...:
  if <condition>:
    wait a second
  stuff here
```
You should use !isFalse instead